spec/messages.js

```diff
@@ -974,11 +974,43 @@ describe("A Chat Message", function () {
         expect(msg.textContent.trim()).toEqual('hello world');
         expect(msg.querySelectorAll('img').length).toEqual(2);
 
+        // @XXX This test isn't really testing anything - needs revisiting
         // Non-https images aren't rendered
         message = base_url+"/logo/conversejs-filled.svg";
         expect(view.content.querySelectorAll('img').length).toBe(4);
         mock.sendMessage(view, message);
         expect(view.content.querySelectorAll('img').length).toBe(4);
+
```

Comment:
The test block above doesn't test anything and needs to be revisited.
Simply put anything in the URL, https or not, and the test will still pass.
I tried to solve it but I'm not exactly sure what setting it is testing.

Reply:
Looks like it also tests for something which isn't implemented anymore. I can't find the ticket where it was asked/decided that images without https shouldn't be rendered, so I've decided to just remove it for now.